r-package/.gitignore

```diff
@@ -1 +1,2 @@
 docs
+geobr.code-workspace
```

Comment:
Why is this necessary?

Reply:
Sorry. My mistake.

Reply:
No problem. try to delete this change from the PR

Reply:
Done